webbpsf/trending.py

```diff
@@ -1502,7 +1498,7 @@ def basic_show_image(image, ax, vmax=0.3, nanmask=1):
                 20, 20, f'{webbpsf.utils.rms(delta_opd, mask=apmask)*1e9:.1f}', color='yellow', fontsize=fs * 0.6
             )
 
-    for i, l in enumerate(['Measured\nWFE', 'Drifts', 'Mirror\nCorrections']):
+    for i, l in enumerate(['Measured\nWFE', 'Drifts', 'Mirror\nCorrections']): # noqa
```

Comment:
why does this need a noqa?

Reply:
the 'l' is ambiguous naming convention

Reply:
ah, yeah. I guess fine to leave as noqa but that's actually a valid critique from the linting. Could refactor that variable name to `label` to avoid the ambiguous single-letter-l.